jekyll/_cci2/artifacts.md

```diff
@@ -131,9 +131,9 @@ Finally, the core dump files are stored to the artifacts service with `store_art
 
 ![Core Dump File in Artifacts Page]( {{ site.baseurl }}/assets/img/docs/core_dumps.png)
 
-When CircleCI runs a build, a link to the core dump file appears under the Artifacts tab of the {% comment %} TODO: Job {% endcomment %}build summary.
+When CircleCI runs a build, a link to the core dump file appears in the Artifacts tab of the **Jobs page**.
 
-## Downloading All Artifacts for a {% comment %} TODO: Job {% endcomment %}Build on CircleCI  
+## Downloading All Artifacts for a Build on CircleCI
```

Comment:
`for a Job on CircleCI`? I may be misreading the TODO comment here.

Reply:
If you look at the URL in the `curl` command below, it looks like it's still pulling info for a specific build number — rather than for a specific job.